src/app/zap-templates/zcl/data-model/chip/push-av-stream-transport-cluster.xml

```diff
@@ -48,8 +38,7 @@ Git: 0.7-summer-2025-1037-g9bd1d5052
     <item name="InvalidStream" value="0x03"/>
     <item name="InvalidURL" value="0x04"/>
     <item name="InvalidZone" value="0x05"/>
-    <item name="UnsupportedContainerFormat" value="0x06"/>
-    <item name="UnsupportedIngestMethod" value="0x07"/>
+    <item name="InvalidCombination" value="0x06"/>
     <item name="InvalidTriggerType" value="0x08"/>
     <item name="InvalidTransportStatus" value="0x09"/>
```

Comment:
I know this matches the spec, but should the spec get renumbered to not skip 7 here, since we haven't shipped this yet?

Reply:
That's fair.
See: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11614